src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/PDFSI.cs

```diff
@@ -152,14 +171,14 @@ private async Task<Stream> GeneratePDF(PDFContext pdfContext)
             return pdfContent;
         }
 
-        private async Task<DataElement> StorePDF(Stream pdfStream, Instance instance, Application appMetadata)
+        private async Task<DataElement> StorePDF(Stream pdfStream, Instance instance, Application appMetadata, string language)
         {
             string fileName = null;
             string app = instance.AppId.Split("/")[1];
 
-            if (!string.IsNullOrEmpty(appMetadata.Title?["nb"]))
+            if (!string.IsNullOrEmpty(appMetadata.Title?[language]))
```

Comment:
Are we sure about this one?? 
If the app is not available in the user's language I'm guessing "nb" will be used, so why not use the "nb" as the default title for the pdf as well? and then a final backup where we use app

Reply:
The logic here is that if we don't find the title for the user language we fall back to a default name. But you want to include a second try for "nb" also? Sure.

Still, I think we should fall back to a default name if no title is defined at all.

Reply:
Yes, I agree. My reasoning behing the suggestions is that Altinn's default language is norwegian. If no title is defined in "nb" thats a mistake on the app owner's side and we handle this by showing the app name.